### DISCUSSION TITLE
Cannot create XML document

### DISCUSSION THREAD

**Discussion by edc1970** [ariacom/Seal-Report#98] [Bug Reports] [answered]
Not quite sure if this is a bug or an installation issue. But I tried a new installation of vesion 8 on my windows machine but immediately upon launching Report Designer and on executing a report, a message popped up saying "There was an error generating the XML document". Subsequent click shows a message saying "Method not found: 'Seal.Model.SealExcelConverter Seal.Model.SealServerConfiguration.get_ExcelConverter()'.

**Comment by ariacom** [comment]
Strange, is it a fresh install or o you had a previous installation (if so which version) ?
In 8.0, the SealServerConfiguration does not have the ExcelConverter property, so it looks like there is an old assembly on your computer.
If you have the full stack trace from the log file (located in <Repository Folder>/Logs, it could be useful to understand.

**Reply by edc1970** [reply]
You are correct, there was a version 6 before I installed this one but I was sure I uninstalled it first. Any advise how to completely uninstall version 6 or any remnants of it?

**Comment by ariacom** [comment, marked as answer]
Just make a backup of you repository folders in case of. Then uninstall/reinstall.
If it still does not work, uninstall, then delete completely the installation directory (e.g. c:\Program Files\Seal Report, this may contains the old assembly !) and the repository folder, finally reinstall.

**Reply by edc1970** [reply]
Completely deleting both installation and repository folders solved the issue. Thank you for your help!
